pom.xml

```diff
@@ -246,13 +264,6 @@
     <dependency>
       <artifactId>workflow-step-api</artifactId>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <optional>true</optional>
```

Comment:
Why is optional removed?

Reply:
There's already a non-optional dependency on `workflow-job`, which itself has a non-optional dependency on `workflow-step-api`. So in practice, the dependency on `workflow-step-api` was already non-optional. Removing `optional` makes this more clear.